cohort-engine/src/main/java/com/ibm/cohort/engine/measure/CDMMeasureEvaluation.java

```diff
@@ -116,48 +116,60 @@ public CDMMeasureEvaluation(DataProvider provider, Interval measurementPeriod) {
 	 * @param measure   CDM Patient Quality Measure
 	 * @param context   CQL Engine Execution Context pre-configured for use in
 	 *                  measure evaluation
-	 * @param patientId Patient ID of the patient to evaluate
+	 * @param patientIds Patient ID(s) of the patient to evaluate (first element when generating INDIVIDUAL type report
 	 * @param evidenceOptions MeasureEvidenceOptions to indicate whether or not to return evaluated resources and define level results
-     * @param parameterMap Map of parameter names to Parameter objects                       	   
+     * @param parameterMap Map of parameter names to Parameter objects
+	 * @param type  Type of report to be generated
 	 * @return MeasureReport with population components filled out.
 	 */
-	public MeasureReport evaluatePatientMeasure(Measure measure, Context context, String patientId, MeasureEvidenceOptions evidenceOptions, Map<String, Parameter> parameterMap) {
+	public MeasureReport evaluatePatientMeasure(Measure measure, Context context, List<String> patientIds, MeasureEvidenceOptions evidenceOptions, Map<String, Parameter> parameterMap, MeasureReport.MeasureReportType type) {
```

Comment:
I'll  bring this up as a topic at a team meeting, but do we need to be mindful of breaking public apis since we have had a `1.0.0` release at this point? Should we aim for backwards compatibility so we don't potentially break consumer code?

Reply:
Talked with team. Don't worry about this change.